### DISCUSSION TITLE
Outdated usage instructions

### DISCUSSION THREAD

**Discussion by renedominik** [community/community#23887] [New to GitHub]
<p>When creating a new repository on the GitHub website, instructions on how to use it are displayed. These still contain:<br>
<code>git remote add origin https://github.com/renedominik/raman.git</code><br>
while the password access via https is deprecated. Shouldn’t this be replaced by the ssh key consistent address?<br>
(I hope this is the right forum.)</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You can still use HTTPS with a <a href="https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token">Personal Access Token</a> instead of a password, so unless the instructions say to use a password they should still be good. I assume they default to HTTPS and not SSH because the former doesn’t require users who aren’t already familiar with SSH to set it up.</p>

**Comment by jsoref** [comment]
<p>While I personally prefer ssh keys, based on the feedback from GitHub’s docs folks, it seems like GitHub actually <em>prefers</em> PATs.</p>
<p>Technically a PAT can be more scoped than a user ssh key (although not as scoped as a deploy key).</p>
